pkgs/development/arduino/arduino-mk/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "1.6.0";
+  name = "arduino-mk-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "sudar";
+    repo   = "Arduino-Makefile";
+    rev    = "${version}";
+    sha256 = "0flpl97d2231gp51n3y4qvf3y1l8xzafi1sgpwc305vwc2h4dl2x";
+  };
+
+  phases = ["installPhase"];
```

Comment:
It is not recommended to set phases directly. I would recommend just putting everything in fetchfromGitHub;

```
fetchFromGitHub rec {
  version = "1.6.0";
	name = "arduino-mk-${version}";
	owner  = "sudar";
	repo   = "Arduino-Makefile";
	rev    = "${version}";
	sha256 = "0flpl97d2231gp51n3y4qvf3y1l8xzafi1sgpwc305vwc2h4dl2x";
}
```

This avoids the need to copy.

Reply:
So, I can easily add the symbolic link stuff. But where are we at ONLY using the `fetchFromGithub`? Because the initial reason was, that it does not include metadata.

Reply:
Resolved as metadata is needed.